nltk/twitter/twitter_demo.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Twitter client
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Ewan Klein <ewan@inf.ed.ac.uk>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Examples to demo the :py:mod:`twitterclient` code.
```

Comment:
Just to clarify: it's fine as far as I'm concerned to move `twitter_demo.py` to the `/test` directory, but I don't think there would be a lot to gain. For reasons mentioned above, it wouldn't make much sense to convert the file to doctest format (we'd have to skip almost all the outputs), so we wouldn't get useful HTML out of it. And we can't use it as a source of automated testing unless the test harness has Twitter API keys available. Going the IPython notebook route still strikes me as the best option.